management/src/main/scala/akka/management/AkkaManagement.scala

```diff
@@ -77,28 +77,33 @@ final class AkkaManagement(implicit system: ExtendedActorSystem) extends Extensi
   private val bindingFuture = new AtomicReference[Future[Http.ServerBinding]]()
   private val selfUriPromise = Promise[Uri]() // TODO has to keep config as well as the Uri, so we can reject 2nd calls with diff uri
 
+  private val providerSettings = {
+    // port is on purpose never inferred from protocol, because this HTTP endpoint is not the "main" one for the app
+    val protocol = if (_connectionContext.isSecure) "https" else "http"
```

Comment:
referring to `_connectionContext` from a `val` makes it impossible to customize this setting `setHttpsContext` - I guess we want to make this a `private def`?

Reply:
:+1: